samples/gamma-fade/main.c

```diff
@@ -0,0 +1,268 @@
+#include <stdint.h>
+#include <stdbool.h>
+#include <string.h>
+#include <math.h>
+#include <hal/video.h>
+#include <hal/debug.h>
+#include <windows.h>
+
+float remap(float in_low,  float in_high,
```

Comment:
Why are some of these functions `static` and others are not? They are only used here, let's just make them `static`

Reply:
Bug, should be fixed.